packages/php-config-printer/tests/NodeFactory/ConstantNodeFactoryTest.php

```diff
@@ -4,52 +4,41 @@
 
 namespace Symplify\PhpConfigPrinter\Tests\NodeFactory;
 
+use PhpParser\Node\Expr\ClassConstFetch;
 use PhpParser\Node\Expr\ConstFetch;
+use PhpParser\Node\Identifier;
+use PhpParser\Node\Name;
 use PHPUnit\Framework\TestCase;
-use Symplify\PhpConfigPrinter\Dummy\YamlContentProvider;
 use Symplify\PhpConfigPrinter\NodeFactory\ConstantNodeFactory;
 
 final class ConstantNodeFactoryTest extends TestCase
 {
     private ConstantNodeFactory $constantNodeFactory;
 
-    private YamlContentProvider $yamlContentProvider;
-
     protected function setUp(): void
     {
-        $this->yamlContentProvider = new YamlContentProvider();
-        $this->constantNodeFactory = new ConstantNodeFactory($this->yamlContentProvider);
+        $this->constantNodeFactory = new ConstantNodeFactory();
     }
 
-    public function testThatDeprecatedPHPConstantExists(): void
-    {
-        $previousLevel = error_reporting(E_ALL & ~E_DEPRECATED);
-
-        $value = constant('PGSQL_LIBPQ_VERSION_STR');
-
-        $this->assertNotEmpty($value);
-
-        error_reporting($previousLevel);
-    }
```

Comment:
Why is this removed?

Reply:
Because it's not needed, this is part of the bruteforcing of constant

Reply:
I vaguelly recall we had a bug reported with missing constant from different PHP version. The class was deprecated and it crashed the tool. Now the tool can deal with those constants too.

Reply:
Also, if you want to make cleanup, it would be easier to review and handle in standalone PR. Here it mixes 2 things - new feature and cleanup, so it makes the review much harder.

Reply:
Sorry but I'm not mixing any thing, this is a refactor which needed a new test case, any cleanup is due to the cognitive complexity rule being enforced

Reply:
So yes after checking this test was not implemented correctly it should have called getConstantValueIgnoringDeprecationWarnings but instead copied the logic, I deleted the method and the test that goes with it

Reply:
Ah, I didn't understand it's related to deleted method `getConstantValueIgnoringDeprecationWarnings()`.
Thanks for explaining.

Allright then :+1: